napari/plugins/_skimage_data.py

```diff
@@ -83,11 +83,19 @@ def _load_skimage_data(name, **kwargs):
     return [(getattr(skimage.data, name)(**kwargs), {'name': name})]
 
 
-@napari_hook_implementation
-def napari_provide_sample_data():
-    from functools import partial
+_DATA = {
+    key: {'data': partial(_load_skimage_data, key), 'display_name': dname}
```

Comment:
@tlambert03 could we wrap the display name with a `trans._(dname)` call?

We have not updated the translation parsing for the manifest files (yet) but this is a good reminder :)

Reply:
these are already wrapped in `trans` above in the `SKIMAGE_DATA` declaration